ddtrace/internal/logger.py

```diff
@@ -177,3 +178,11 @@ def handle(self, record):
             # Increment the count of records we have skipped
             # DEV: `self.buckets[key]` is a tuple which is immutable so recreate instead
             self.buckets[key] = DDLogger.LoggingBucket(logging_bucket.bucket, logging_bucket.skipped + 1)
+
+
+def _get_handler(logger: logging.Logger, handler_name: str) -> Optional[logging.Handler]:
```

Comment:
Originally I had this type hint as `DDLogger`, but looks like the `ddtrace` logger is a `logging.Logger` type and not `DDLogger`? Not sure if this is intended

Reply:
I think from the perspective of type hints' static analysis, it's just a `Logger` because it is passed through `logging.getLogger`.